packages/vite-plugin-cloudflare/src/workers-configs.ts

```diff
@@ -351,8 +352,74 @@ export function getWorkerConfig(
 	};
 }
 
+/**
+ * Returns the path to a wrangler config for a worker after having it validated
+ * (throws appropriate errors in case the validation fails)
+ *
+ * @param root the root of the vite project
+ * @param requestedConfigPath the requested config path, if any
+ * @param isForAuxiliaryWorker whether the config path is being requested for an auxiliary worker
+ * @returns a valid path to a config file
+ */
+export function getValidatedWranglerConfigPath(
+	root: string,
+	requestedConfigPath?: string,
+	isForAuxiliaryWorker = false
+) {
+	if (requestedConfigPath) {
+		const configPath = path.resolve(root, requestedConfigPath);
+
+		const forAuxiliaryWorkerErrorMessage = isForAuxiliaryWorker
+			? " requested for one of your auxiliary workers"
+			: "";
+
+		const errorMessagePrefix = `The provided configPath (${configPath})${forAuxiliaryWorkerErrorMessage}`;
+
+		const fileExtension = path.extname(configPath).slice(1);
+		const allowedExtensions = ["jsonc", "json", "toml"];
```

Comment:
Let's extract this and also use it in `findWranglerConfig`.

Reply:
oh yeah of course, thanks!